preppal-fe/src/pages/recipe.tsx

```diff
@@ -0,0 +1,85 @@
+import { Container, Image, Stack, Row, Col, Card} from 'react-bootstrap';
+import {FaStar, FaRegStar, FaHeart} from 'react-icons/fa';
+const chickenfriedrice = require('../assets/chickenfriedrice.webp')
+
+export default function Recipe() {
+    return (
+        <Container className="d-flex ">
+        <html>
+            <h1>Chicken's Fried Rice</h1>
```

Comment:
We should replace all of these hard-coded strings into a variable, so that when we connect it, it should load dynamically according to all the information